<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="handleWidth">
       <number>2</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <underline>true</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="elideMode">
        <enum>Qt::ElideLeft</enum>
       </property>
       <property name="documentMode">
        <bool>true</bool>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <property name="tabBarAutoHide">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tab">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Show the Settings&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>60</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
        <widget class="QSlider" name="horizontalSlider">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>190</y>
           <width>291</width>
           <height>22</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QProgressBar" name="progressBar">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>150</y>
           <width>321</width>
           <height>23</height>
          </rect>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
        <widget class="QProgressBar" name="progressBar2">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>380</y>
           <width>301</width>
           <height>23</height>
          </rect>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab3">
        <attribute name="title">
         <string>Page</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab4">
        <attribute name="title">
         <string>Page</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>956</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionEdit"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionConnect_Debugger"/>
    <addaction name="actionDisconnect_Debugger"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDebug"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionRun_All"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>:/Resources/Resources/option-icon.png</normaloff>:/Resources/Resources/option-icon.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Resources/Resources/Íffnen.bmp</normaloff>:/Resources/Resources/Íffnen.bmp</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Resources/Resources/save-48x48.png</normaloff>:/Resources/Resources/save-48x48.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset>
     <normaloff>:/Resources/Resources/option-icon.png</normaloff>:/Resources/Resources/option-icon.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>:/Resources/Resources/cancel2-24x24.png</normaloff>:/Resources/Resources/cancel2-24x24.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionConnect_Debugger">
   <property name="icon">
    <iconset>
     <normaloff>:/Resources/Resources/faq-icon.png</normaloff>:/Resources/Resources/faq-icon.png</iconset>
   </property>
   <property name="text">
    <string>Connect Debugger</string>
   </property>
  </action>
  <action name="actionDisconnect_Debugger">
   <property name="icon">
    <iconset>
     <normaloff>:/Resources/Resources/btnoff-32x32.png</normaloff>:/Resources/Resources/btnoff-32x32.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect Debugger</string>
   </property>
  </action>
  <action name="actionRun_All">
   <property name="text">
    <string>Run All</string>
   </property>
   <property name="toolTip">
    <string>Run All</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
